src/wallet/coincontrol.h

```diff
@@ -44,20 +44,15 @@ class CCoinControl
     //! Avoid partial use of funds sent to a given address
     bool m_avoid_partial_spends;
```

Comment:
Could be default initialised to `DEFAULT_AVOIDPARTIALSPENDS`?

Reply:
or made `const`, which also avoids uninitialized reads

Reply:
It's changed in wallet.cpp.

Reply:
Initialised to `DEFAULT_AVOIDPARTIALSPENDS`.